examples/aws_container_app/main.tf

```diff
@@ -22,6 +22,8 @@ module "port_ocean_ecs" {
   cluster_name                          = var.cluster_name
   existing_cluster_arn                  = var.existing_cluster_arn
   account_list_regions_resources_policy = var.account_list_regions_resources_policy
+  vpc_id = var.vpc_id
+  create_default_sg = var.create_default_sg
```

Comment:
shouldn't this be added as part of the variables inside this directory as well?

Reply:
They are already part of the "Global" variables